_data/opencollective.json

```diff
@@ -26,7 +26,7 @@
   {
     "name": "MailerToGo",
     "url": "https://MailerToGo.com",
-    "logo": "sponsors/mailertogo.svg",
```

Comment:
This change is odd... we override to `svg` in `others.json`:
https://github.com/crystal-lang/crystal-website/blob/cd48569a5cfd477abdb828e4d3e8398a907b0f42/_data/others.json#L30

Reply:
It's even the `svg` in `_data/sponsors.json` below...